packages/ag-charts-enterprise/src/features/zoom/zoomToolbar.ts

```diff
@@ -103,8 +102,8 @@ export class ZoomToolbar extends BaseProperties {
         this.destroyFns.push(
             this.toolbar.addToolbarListener('button-pressed', this.onButtonPress.bind(this)),
             this.toolbar.addToolbarListener('button-focused', this.onButtonFocus.bind(this)),
-            ctx.interactionManager.addListener('hover', this.onHover.bind(this), InteractionState.All),
-            ctx.interactionManager.addListener('leave', this.onLeave.bind(this), InteractionState.All),
+            ctx.domManager.containerWidget.addListener('mousemove', this.onHover.bind(this)),
+            ctx.domManager.containerWidget.addListener('mouseleave', this.onLeave.bind(this)),
```

Comment:
Could these be listeners added to the `root` region?

Reply:
The `'root'` region only fires `'hover'` events when hovering directly above the container (i.e. root) element, it does not bubble and therefore does not receive `'hover'` events when hovering over the series-area. As a consequence, the conditions in the `onHover` handler that determine the visibility state will always be `false`, because the zoom toolbar can only become visible when hovering in the series-area.

In theory, it could be added to the `'series'` and the `event.canvasY` value could be used to determine whether the zoom toolbar should be visible. However, this will fire a `'leave'` event when hovering over a zoom button, which toggles the visibility back to `false`, causing the zoom toolbar to wiggle up & down:

https://github.com/user-attachments/assets/f6020655-8d95-4a64-8d50-a8386bcb8692

In the past, we've addressed this sort of problem by adding a condition to `shouldIgnore`, but we want to move away from the approach because it's difficult to maintain.
https://github.com/ag-grid/ag-charts/blob/aa3e0f1bfe492cb0c1e2b86142895f5633748e94/packages/ag-charts-community/src/chart/interaction/regionManager.ts#L85

I initially tried adding event listeners to the `seriesWidget` instead. The issue with this however is that the zoom toolbar HTML elements are not a descendant of the `seriesWidget` element, so this still fires `'mouseleave'` events when hovering over a zoom button and produces the same affect as in this video above.

Adding a listener to the `containerWidget` ensures that `'mouseleave'` is only fired when the mouse completely exits the bounds of the entire chart, and always fire `'mousemove'` event within the chart to ensure that the correct visibility state is computed by `onHover`.